$MirOS: ports/net/centericq/patches/patch-kkconsui-0_1_src_cmenus_cc,v 1.1.7.1 2005/03/18 15:48:03 tg Exp $
$OpenBSD: patch-kkconsui-0_1_src_cmenus_cc,v 1.6 2003/12/23 21:08:29 pvalchev Exp $
--- kkconsui-0.1/src/cmenus.cc.orig	Sun Mar 28 11:38:13 2004
+++ kkconsui-0.1/src/cmenus.cc	Tue Jul  5 15:14:01 2005
@@ -252,6 +252,7 @@ int verticalmenu::open() {
 		    return 0;
 		    break;
 	
+		case 'k':
 		case KEY_UP:
 		    if(curelem > 0) {
 			shownelem(curelem, 0);
@@ -282,6 +283,7 @@ int verticalmenu::open() {
 		    }
 		    break;
 
+		case 'j':
 		case KEY_DOWN:
 		    if(!items.empty()) {
 			if(curelem < items.size()-1) {
